### ISSUE TITLE
Align doesn't work as expected

### ISSUE THREAD

**Issue by tomi** [steve8708/vscode-align#4]
I have this code I want to align:

``` javascript
fromCell: selectedRange.fromCell,
fromRow: selectedRange.fromRow,
toCell: selectedRange.fromCell + clipArea.destW - 1,
toRow: selectedRange.fromRow + clipArea.destH - 1
```

I would expect it to be aligned like this:

``` javascript
fromCell: selectedRange.fromCell,
fromRow:  selectedRange.fromRow,
toCell:   selectedRange.fromCell + clipArea.destW - 1,
toRow:    selectedRange.fromRow + clipArea.destH - 1
```

but instead the extension aligns it like this:

``` javascript
        fromCell:      selectedRange.fromCell,
        fromRow:       selectedRange.fromRow,
toCell: selectedRange. fromCell + clipArea.destW - 1,
toRow:  selectedRange. fromRow + clipArea.destH - 1
```

I'm using version 0.0.6

**Comment by bcrotty** [comment]
I'm also having issues. When I try to align the following PowerShell code:

```
$TotalSteps = 6
$Activity = "Migrating Users"
$Status = '"Step $Step of $TotalSteps | $StepText"'
$StatusBlock = [ScriptBlock]::Create($Status)
$Id = 1
```

I expect to see:

```
$TotalSteps  = 6
$Activity    = "Migrating Users"
$Status      = '"Step $Step of $TotalSteps | $StepText"'
$StatusBlock = [ScriptBlock]::Create($Status)
$Id          = 1
```

But instead I see:

```
$TotalSteps  = 6
$Activity    = "Migrating Users"
$Status      = '"Step $Step of $TotalSteps | $StepText"'
$StatusBlock = [ScriptBlock]:                             :Create($Status)
$Id          = 1
```

I'm also using version 0.0.6.

**Comment by steve8708** [comment]
Fixed via  #6
